make/tests.mk

```diff
@@ -43,6 +43,13 @@ always-run:
 # Fully qualified device name
 FQDN = $(ARCH)-$(DEVICE_TYPE)-$(DEVICE)
 
+TEST_DIR = $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
+OUT_LOCAL = $(TEST_DIR)/build-$(FQDN)
```

Comment:
Should this be defined in `$(TOP_DIR)/$(ARCH)/make/tests.mk` file then?

Reply:
There is a interdependence now so I don't that would work. I think we could split the arch into a pre and post mk files and it would be cleaner.